lib/bolt/cli.rb

```diff
@@ -463,9 +463,9 @@ def execute(options)
       when 'module'
         case options[:action]
         when 'add'
-          code = add_project_module(options[:object], config.project)
+          code = add_project_module(options[:object], config.project, options[:force])
         when 'install'
-          code = install_project_modules(config.project, options[:force])
+          code = install_project_modules(config.project, options[:force], options[:resolve])
```

Comment:
This results in `bolt module install` never resolving module declarations, as `resolve` will be `nil` unless you use the `--[no]-resolve` flag. Since you're explicitly passing `nil` as a parameter, `Bolt::ModuleInstaller.install` will not default to `true`.